CHANGELOG.md

```diff
@@ -9,13 +9,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 The minor version will be incremented upon a breaking change and the patch version will be
 incremented for features.
 
+## [Unreleased Breaking]
```

Comment:
Note that you can simply use `###Breaking` and place it under the `##[Unreleased]` section.